rpc_jobs/jjb_setup.yml

```diff
@@ -29,45 +45,39 @@
                   )
                 ]){
                   lock('jjb'){
-                  sh """#!/bin/bash -xe
-                    source ../.venv/bin/activate
+                    dir("jenkins_jobs"){
+                      sh """#!/bin/bash -xe
+                        source .venv/bin/activate
 
-                    cat > jenkins_jobs.ini << EOF
-      [job-builder]
-      ignore_cache=False
-      keep_descriptions=False
-      include_path=.:scripts:~/git/
-      recursive=True
-      exclude=.*manual:./development
-      allow_deplicates=False
+                        rpc-gating/scripts/jenkins-jobs-re.py \
+                          jobSources.collect {--job-source "${it.repo}:${it.commitish}"} \
+                          --jenkins-url '${JENKINS_URL}' > jenkins_jobs.ini
 
-      [jenkins]
-      url=${JENKINS_URL}
-      EOF
 
-                    if [[ "$RPC_GATING_BRANCH" == "master" ]] && [[ "$JOBS" == "-r rpc_jobs" ]]; then
-                        echo "Setting --delete-old as RPC_GATING_BRANCH=master and JOBS='-r rpc_jobs'"
-                        UPDATE_ARGS="--delete-old"
-                    fi
+                        if [[ "${deleteOldJobs}" == "false ]]; then
```

Comment:
Should this be checking for ``true``?

Reply:
That line is nonsense and broken in multiple ways, this pull request is not ready for review.